JASP-Engine/JASP/R/ttestbayesianonesample.R

```diff
@@ -374,11 +374,23 @@
 	
 	points(0, .dprior(0,r, oneSided= oneSided), col="black", pch=21, bg = "grey", cex= cexPoints)
 	
+	if (oneSided == FALSE) {
+	
+		heightPosteriorAtZero <- .dposteriorShiftedT(0, parameters=parameters, oneSided=oneSided)
+	
+	} else if (oneSided == "right") {
+	
+		posteriorLineLargerZero <- posteriorLine[posteriorLine > 0]
+		heightPosteriorAtZero <- posteriorLineLargerZero[1]
+		
+	} else if (oneSided == "left") {
+	
+		posteriorLineLargerZero <- posteriorLine[posteriorLine > 0]
+		heightPosteriorAtZero <- posteriorLineLargerZero[length(posteriorLineLargerZero)]
+	}
 	
```

Comment:
quick comment; it's good to avoid assigning different types to the same variable. no time to explain!